File: chat-uikit/src/main/java/io/agora/chat/uikit/chat/interfaces/EaseEmojiconMenuListener.java
package io.agora.chat.uikit.chat.interfaces;


public interface EaseEmojiconMenuListener{
        /**
         * on emojicon clicked
         * @param emojicon
         */
        default void onExpressionClicked(Object emojicon){}
        /**
         * on delete image clicked
         */
        default void onDeleteImageClicked() {}

        /**
         * On send icon clicked
         */
        default void onSendIconClicked() {}

        default void onTabBarItemClick(int index){}
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/feature/chat/interfaces/EaseEmojiconMenuListener.kt
package io.agora.uikit.feature.chat.interfaces

interface EaseEmojiconMenuListener {
    /**
     * on emojicon clicked
     * @param emojiIcon
     */
    fun onExpressionClicked(emojiIcon: Any?)

    /**
     * on delete image clicked
     */
    fun onDeleteImageClicked() {}

    /**
     * On send icon clicked
     */
    fun onSendIconClicked() {}
}